tests/virtualization/vagrant/boxes/tumbleweed.pm

```diff
@@ -80,4 +80,8 @@ sub run() {
     assert_script_run("popd");
 }
 
+sub post_fail_hook() {
+    upload_logs($vagrant_logfile);
```

Comment:
Missing `$self->SUPER::post_fail_hook;`, also in the other module (before this PR already...)

Reply:
Needs `my ($self) = @_;`

Reply:
Needs something like `my ($self) = @_;`